### ISSUE TITLE
pagination with number of pages

### ISSUE THREAD

**Issue by dvlopjs** [CS2Browser/cs2-server-browser#66]
I think it is a simple feature that adds value to the UX.
![image](https://github.com/user-attachments/assets/1246e18d-6772-40d6-a6c4-f19fa45398a1)

**Comment by JMorell** [comment]
I'll think about it, but it's expensive to have the total page count every time you load the listing.

**Comment by dvlopjs** [comment]
Yes, it is true, perhaps caching the total can be done without having such an impact.
